test/games/prime1/exporter/test_prime_game_exporter.py

```diff
@@ -5,19 +5,27 @@
 import pytest
 
 from randovania.games.prime1.exporter.game_exporter import PrimeGameExporter, PrimeGameExportParams, adjust_model_names
+from randovania.games.prime1.layout.prime_configuration import EnemyAttributeRandomizer
 
 
 @pytest.mark.parametrize('use_echoes_models', [True, False])
-def test_patch_game(mocker, tmp_path, use_echoes_models):
+@pytest.mark.parametrize('use_enemy_attribute_randomizer', [False, True])
+def test_patch_game(mocker, tmp_path, use_echoes_models, use_enemy_attribute_randomizer):
     mock_symbols_for_file: MagicMock = mocker.patch("py_randomprime.symbols_for_file", return_value={
         "UpdateHintState__13CStateManagerFf": 0x80044D38,
     })
+    seed = 103817502
+    if use_enemy_attribute_randomizer:
+        enemy_attribute_randomizer = EnemyAttributeRandomizer(1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, False).as_json
```

Comment:
Why are you testing with values that do nothing? Shouldn't you test with values that actually make game modifications?

Reply:
Ahh right, was focused on the other aspects of the test.